### PR TITLE
Ran black, updated to pylint 2.x

### PR DESCRIPTION
**Pull request by evaherrada**
I replaced a comprehension with a list constructor in line 330/331 of adafruit_drv2605. I _believe_ this should work the same based on some testing in the Linux REPL, but I'd prefer if someone could test it with the hardware.

### CODE REVIEW

File: adafruit_drv2605.py [hunk 1]
```diff
@@ -327,4 +328,4 @@ def __iter__(self):
 
     def __repr__(self):
         """Return a string representation of all slot's effects."""
-        return repr([effect for effect in self])
+        return repr(list(effect for effect in self))
```
Comment (kattni) [inline adafruit_drv2605.py]: @dherrada Why was that change required? Using `list()` isn't as good performance-wise, and isn't really as Pythonic.

**Review by kattni — CHANGES_REQUESTED** [review]

File: adafruit_drv2605.py (hunk 1 above)
Comment (tannewt) [reply adafruit_drv2605.py]: Black or lint has a validator for unnecessary list comprehensions like this. Comprehensions should only be used when mutating each element (aka more than just `effect` at the start.)

The simplest way to change the `self` object to a list is `list(self)`.

```suggestion
        return repr(list(self))
```

**Review by kattni — APPROVED** [review]
